package.json

```diff
@@ -125,6 +125,7 @@
     "ngx-modal": "0.0.29",
     "ngx-widgets": "0.14.0",
     "offline-plugin": "4.6.2",
+    "protractor-jasmine2-screenshot-reporter": "0.3.5",  
```

Comment:
this should be in the next section for devDependencies as it is not used in the running app.